api/api/admin/media_report.py

```diff
@@ -0,0 +1,199 @@
+import logging
+
+from django.conf import settings
+from django.contrib import admin
+from django.urls import reverse
+from django.utils.safestring import mark_safe
+
+from elasticsearch import NotFoundError
+from elasticsearch_dsl import Search
+
+from api.models import PENDING
+
+
+def create_link(url: str, text: str):
+    return mark_safe(f'<a href="{url}">{text}</a>')
+
+
+class MediaReportAdmin(admin.ModelAdmin):
+    change_form_template = "admin/api/media_report/change_form.html"
+    list_display = ("id", "reason", "is_pending", "description", "created_at", "url")
+    list_filter = (
+        ("decision", admin.EmptyFieldListFilter),  # ~status, i.e. pending or moderated
+        "reason",
+    )
+    list_display_links = ("id",)
+    list_select_related = ("media_obj",)
+    search_fields = ("description", "media_obj__identifier")
+    autocomplete_fields = ("media_obj",)
+    actions = None
+    media_type = None
+
+    @admin.display(description="Has sensitive text")
+    def has_sensitive_text(self, obj):
+        """
+        Return `True` if the item cannot be found in the filtered index - which means the item
+        was filtered out due to text sensitivity.
+        """
+        if not self.media_type or not obj:
+            return None
+
+        filtered_index = f"{settings.MEDIA_INDEX_MAPPING[self.media_type]}-filtered"
+        try:
+            search = (
+                Search(index=filtered_index)
+                .query("term", identifier=obj.media_obj.identifier)
+                .execute()
+            )
+            if search.hits:
+                return False
+        except NotFoundError:
+            logging.error(f"Could not resolve index {filtered_index}")
+            return None
+        return True
+
+    @admin.display(description="URLs")
+    def landing_urls(self, obj):
+        if not self.media_type or not obj:
+            return "N/A"
+        openverse_url = (
+            f"https://openverse.org/{self.media_type}/{obj.media_obj.identifier}"
+        )
+        return mark_safe(
+            f"<p>{create_link(obj.media_obj.foreign_landing_url, obj.media_obj.provider)}, "
+            f'{create_link(openverse_url, "openverse.org")}'
+        )
+
+    def get_other_reports(self, obj):
+        if not self.media_type or not obj:
+            return []
+
+        reports = (
+            self.model.objects.filter(media_obj__identifier=obj.media_obj.identifier)
+            .exclude(id=obj.id)
+            .order_by("created_at")
+        )
+        for report in reports:
+            report.href = reverse("admin:api_imagereport_change", args=[report.id])
```

Comment:
I believe this will also need to be reversed by media type:

```suggestion
            report.href = reverse(f"admin:api_{self.media_type}report_change", args=[report.id])
```